components/NewsAndMedia.js

```diff
@@ -5,20 +5,32 @@ import Title from './Title'
 function NewsAndMedia() {
   return (
     <section className="container py-5 d-flex flex-column justify-content-center align-items-center">
-      <Title
-        title="News and Media"
-        position="middle"
-      />
+      <Title title="News and Media" position="middle" />
       <div className="row">
-        <div className="col-12 col-md-7" style={{ width: "700", height: "400px", border: "2px solid black" }}>
-          <h1> Instagram </h1> 
+        <div
+          className="col-12 col-md-7"
+          style={{ width: '700', height: '400px', border: '2px solid black' }}
+        >
+          <h1> Instagram </h1>
         </div>
         <div className="col-12 col-md-5">
-          <a class="twitter-timeline" data-width="700" data-height="400" href="https://twitter.com/resilient_u?ref_src=twsrc%5Etfw">Tweets by resilient_u</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+          <a
+            className="twitter-timeline"
+            data-width="700"
+            data-height="400"
+            href="https://twitter.com/resilient_u?ref_src=twsrc%5Etfw"
+          >
+            Tweets by resilient_u
+          </a>{' '}
+          <script
+            async
+            src="https://platform.twitter.com/widgets.js"
+            charSet="utf-8"
+          ></script>
```

Comment:
go to https://msmr-git-staff.cache-and-cookies.vercel.app/about

then click on top left 
![image](https://user-images.githubusercontent.com/47992694/108605615-475b2e00-7383-11eb-80e8-5f24ca6de720.png)

then scroll down to this twitter component and this happens

![image](https://user-images.githubusercontent.com/47992694/108605633-593cd100-7383-11eb-941d-e13704e035ac.png)


I have reproduced this bug multiple times, msg me on discord if you need help trying to reproduce it. Not sure if it was in previous version

Reply:
UPDATE: 

Same Issue is on development branch too

https://msmrdev.vercel.app/

Reply:
@jimmyygit this bug was in the previous commit from you

Reply:
ok I see. I can reproduce it too. There seems to be an issue with the twitter content not loading when user goes to `\` from somewhere else.